tests/Backend/Snowflake/WorkspacesLoginNameTest.php

```diff
@@ -0,0 +1,62 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Keboola\Test\Backend\Snowflake;
+
+use Generator;
+use Keboola\TableBackendUtils\Escaping\Snowflake\SnowflakeQuote;
+use Keboola\Test\Backend\WorkspaceConnectionTrait;
+use Keboola\Test\Backend\WorkspaceCredentialsAssertTrait;
+use Keboola\Test\Backend\Workspaces\ParallelWorkspacesTestCase;
+
+class WorkspacesLoginNameTest extends ParallelWorkspacesTestCase
+{
+    use WorkspaceConnectionTrait;
+    use WorkspaceCredentialsAssertTrait;
+
+    /**
+     * @return Generator<string, array{async:bool}>
+     */
+    public static function syncAsyncProvider(): Generator
+    {
+        yield 'sync' => [
+            'async' => false,
+        ];
+        yield 'async' => [
+            'async' => true,
+        ];
+    }
+
+    /**
+     * @dataProvider syncAsyncProvider
+     */
+    public function testWorkspaceCreate(bool $async): void
+    {
+        $this->allowTestForBackendsOnly(
```

Comment:
btw proc tady mame toto? je to Test classa ve slozce Snowflake, s namespace snowflake . kdyz to nekdo bude poustet na BQ, tak je proste ocas

Reply:
smazal sem